incubator/hnc/internal/validators/hncconfig.go

```diff
@@ -211,14 +217,10 @@ func (a ancestorObjects) copy() ancestorObjects {
 	return copy
 }
 
-func (a ancestorObjects) add(onm string, ns *forest.Namespace) {
+func (a ancestorObjects) add(onm string, ns *forest.Namespace, nsLabels labels.Set) {
```

Comment:
You're not doing anything with `nsLabels`, did you mean to?

Reply:
Forgot to remove it, done